SwiftLint/swiftlint.yml

```diff
@@ -253,6 +253,16 @@ custom_rules:
     regex: '(?-smxi)(func get[A-Z0-9][\w]+?(\())'
     message: "You should not prefix your functions with the word get."
     severity: warning
+  body_indentation:
+    name: "Body Indentation"
+    regex: '(?m-six)^(?<tabs>[\t]+)(if|guard|case)\n[^{]+[\{]\n+\k<tabs>{1}[\t]{1}[\w]'
+    message: "You should indent the body by one level"
+    severity: warning
+  condition_split:
```

Comment:
The name is quite generic for the the specific rule. Something like `one_condition_per_if_statement_line` would avoid conflicts to other rules which we or SwiftLint might add regarding condition split